server/schema/submissions/createSubmission.js

```diff
@@ -0,0 +1,24 @@
+const schema = {
+  required: ['user_submission', 'output'],
+  properties: {
+    user_submission: {
+      type: 'string',
+    },
+    output: {
+      type: 'string',
+    },
+  },
+  errorMessage: {
+    required: {
+      user_submission: 'User submission required',
+      output: 'Output required',
+    },
+    properties: {
+      user_submission: 'Invalid user submission',
+      output: 'Invalid output',
+    },
+    _: 'Invalid data',
+  },
+}
+
+module.exports = schema
```

Comment:
I will change the sql accordingly, so now `user_submission` field will be removed and  two new fields will be added `subjective_submission` & `mcq_submission`. So this should probably be like this

const schema = {
  required: ['output'],
  oneOf: [
    required: ['subjective_submission'],
    required: ['mcq_submission'],
  ],
  properties: {
    subjective_submission: {
      type: 'string',
    },
    mcq_submission: {
      type: 'integer',
    },
    output: {
      type: 'string',
    },
  },
  errorMessage: {
    required: {
      user_submission: 'User submission required',
      output: 'Output required',
    },
    properties: {
      user_submission: 'Invalid user submission',
      output: 'Invalid output',
    },
    _: 'Invalid data',
  },
}